group.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+type Group struct {
```

Comment:
Don't capitalize the name of structs, constants etc. if it not really necessary. In this case, you have a package `main` only which won't be imported, so you don't need any exported struct/variable.